### PR TITLE
feat: beyond progressive web apps part 3

### PR DESCRIPTION
**Pull request by janl**
blog post version of https://github.com/hoodiehq/hood.ie/issues/303 — would love a final review by someone, but also would like to post soon 😁

### CODE REVIEW

**Comment by janl** [comment]
Preview here: https://github.com/hoodiehq/hood.ie/blob/f97e4b7ba35ec97d93e57aad27d20eeb3cd53a02/_posts/2016-06-22-beyond-progressive-web-apps-part-3.md 👀

**Comment by espy** [comment]
LGTM

File: _posts/2016-06-22-beyond-progressive-web-apps-part-3.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+![Lt. Frank Columbo, LAPD, Homicide](/dist1/blog/2016/06/columbo.jpg)
+
+Remember when we talked about the author and the editor of a story fixing a typo and adding to the story at the same time? We now know to detect and how to resolve this conflict.
+
+But what if the typo fix both create result in the *same* text? With timestamps or vector clocks, a conflict is still generated, and our subsequent merge procedure has got nothing to do. So we can handle this too, but since the resulting story is the same, wouldn’t it be nice to *not* create a conflict here in the first place?
+
+I realise this example is a little bit contrived, but I hope by staying within our example project, we can see the problem here. Where the problem would *actually* make our lives harder is when we don’t have a single sync server, but a cluster of sync servers, which we might need for scaling to higher and higher loads of our system.
+
+Explaining all this takes time, and this tutorial is already getting long, so we are going to skip the nitty-gritty details of that one. I hope you forgive me. If you don’t, I hope this gif of a squirrel makes up for it:
```
Comment (NickColley) [inline _posts/2016-06-22-beyond-progressive-web-apps-part-3.md:132]: Wondering if maybe a simpler way to say that? https://www.theguardian.com/uk/2002/may/15/race.world

Not sure but thought worth mentioning ha

Comment (janl) [inline _posts/2016-06-22-beyond-progressive-web-apps-part-3.md:132]: only saw this after hitting the merge button, fixed on main branch, thanks for the heads up!

**Pull request merged**